src/sidebar/components/test/annotation-omega-test.js

```diff
@@ -114,4 +139,171 @@ describe('AnnotationOmega', () => {
+    it('should not show the publish control if not in edit mode', () => {
+      setEditingMode(false);
+
+      const wrapper = createComponent();
+
+      assert.notOk(wrapper.find('AnnotationPublishControl').exists());
+    });
+
+    it('should set the publish control to disabled if annotation is empty', () => {
+      const draft = fixtures.defaultDraft();
+      draft.tags = [];
+      draft.text = '';
+      fakeStore.getDraft.returns(draft);
+
+      const wrapper = createComponent();
+
+      assert.isTrue(
+        wrapper.find('AnnotationPublishControl').props().isDisabled
+      );
+    });
+
+    it('should set `isShared` to `false` if annotation is private', () => {
+      const draft = fixtures.defaultDraft();
+      draft.isPrivate = true;
+
+      fakeStore.getDraft.returns(draft);
+
+      const wrapper = createComponent();
+
+      assert.isFalse(wrapper.find('AnnotationPublishControl').props().isShared);
+    });
+
+    it('should set `isShared` to `true` if annotation is shared', () => {
+      const draft = fixtures.defaultDraft();
+      draft.isPrivate = false;
+      fakeStore.getDraft.returns(draft);
+
+      const wrapper = createComponent();
+
+      assert.isTrue(wrapper.find('AnnotationPublishControl').props().isShared);
+    });
+
+    it('should update annotation privacy when changed by publish control', () => {
+      setEditingMode(true);
+
+      const wrapper = createComponent();
+
+      act(() => {
+        wrapper
+          .find('AnnotationPublishControl')
+          .props()
+          .onSetPrivacy({ level: 'private' });
+      });
+
+      const call = fakeStore.createDraft.getCall(0);
+
+      assert.calledOnce(fakeStore.createDraft);
+      assert.isTrue(call.args[1].isPrivate);
+    });
+
+    it('should update annotation privacy default on change', () => {
+      setEditingMode(true);
+
+      const wrapper = createComponent();
+
+      act(() => {
+        wrapper
+          .find('AnnotationPublishControl')
+          .props()
+          .onSetPrivacy({ level: 'private' });
+      });
+
+      assert.calledOnce(fakeStore.setDefault);
+      assert.calledWith(fakeStore.setDefault, 'annotationPrivacy', 'private');
+    });
+
+    it('should not update annotation privacy default on change if annotation is reply', () => {
+      fakeMetadata.isReply.returns(true);
+
+      setEditingMode(true);
+
+      const wrapper = createComponent();
+
+      act(() => {
+        wrapper
+          .find('AnnotationPublishControl')
+          .props()
+          .onSetPrivacy({ level: 'private' });
+      });
+
+      assert.equal(fakeStore.setDefault.callCount, 0);
+    });
+  });
+
+  describe('license information', () => {
+    it('should show license information when editing shared annotations in public groups', () => {
+      fakeStore.getGroup.returns({ type: 'open' });
+      setEditingMode(true);
+
+      const wrapper = createComponent();
+
+      assert.isOk(wrapper.find('AnnotationLicense').exists());
```

Comment:
Unless you are intentionally wanting to check truthiness/falsiness vs true/false, I prefer to use `isTrue` and `isFalse` rather than `isOk` and `notOk` respectively, as this is more explicit about what result is expected.